src/threads/openThread.ts

```diff
@@ -0,0 +1,18 @@
+import type { SocketHandler } from '../utils/types';
+import { ForumTable } from './table';
+import { extractTokenData } from '../parsers';
+import { dynamodb } from '../dynamodb';
+
+const forumTable = new ForumTable(dynamodb);
+
+export const handler: SocketHandler = async event => {
+  const tokenData = extractTokenData(event);
```

Comment:
Probably it would be cleaner to do the token extraction everywheere using a middleware approach (maybe https://www.serverless.com/plugins/serverless-middleware may help). But it can clearly be done later.